nifi-frontend/src/main/frontend/package.json

```diff
@@ -44,46 +45,47 @@
         "zone.js": "0.14.4"
     },
     "devDependencies": {
-        "@angular-builders/jest": "^17.0.2",
-        "@angular-devkit/build-angular": "17.2.0",
-        "@angular-devkit/core": "~17.1.3",
-        "@angular-devkit/schematics": "~17.1.3",
+        "@angular-builders/jest": "^18.0.0",
+        "@angular-devkit/build-angular": "18.0.7",
+        "@angular-devkit/core": "18.0.7",
         "@angular-eslint/builder": "17.2.1",
```

Comment:
Is this dependency needed? If yes, should it have been updated too? It was the source of a npm installation warning.